src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/services/ConfigureTaskService.java

```diff
@@ -37,6 +39,7 @@ public ConfigureTaskService(CamundaService camundaService,
     public void configureTask(String taskId) {
 
         CamundaTask task = camundaService.getTask(taskId);
+        log.info("Task details retrieved from Camunda : {}", task);
```

Comment:
Being picky again but instead of printing the entire object can we print only the CamundaTask Id the rest is kind of noise since we only make use of 3 things from this object:
name id and due date I think